tests/test_wsfexv1.py

```diff
@@ -39,15 +40,20 @@
 
 # obteniendo el TA para pruebas
 
-ta = WSAA().Autenticar("wsfex", "reingart.crt", "reingart.key")
-print(ta)
 
+pytestmark =pytest.mark.vcr
+
+@pytest.fixture(scope='module')
+def vcr_cassette_dir(request):
+    # Put all cassettes in vhs/{module}/{test}.yaml
+    return os.path.join('tests/cassettes', request.module.__name__)
 
 class TestFEX(unittest.TestCase):
```

Comment:
Please create a ticket to move this test to pytest (like you did in the other classes)